test/filter_design.jl

```diff
@@ -364,8 +363,19 @@ end
         -0.9969173244841298-0.07845911266921719im,
         -0.97236994351794+0.2334453500964366im,
         -0.97236994351794-0.2334453500964366im]
-    # println(ComplexF64([sort(f.p, lt=lt) - sort(Butterworth(BigFloat, 20).p, lt=lt) sort(m_p, lt=lt) - sort(Butterworth(BigFloat, 20).p, lt=lt)]))
-    zpkfilter_accuracy(f, ZeroPoleGain(Float64[], m_p, 1), Butterworth(BigFloat, 20); eps=1e-6, compare_gain_at=0, relerr=2)
+
+    # compare frequency responses to a reference
+    freq = [0; 10 .^ range(big(-2), 2, length=10000)]
+    H_ref = freqresp(Butterworth(BigFloat, 20), freq)
+    H_1 = freqresp(f, freq)
+    H_2 = freqresp(ZeroPoleGain{:s}(Float64[], m_p, 1), freq)
+    # both should be good
+    @test H_1 ./ H_ref ≈ ones(length(freq))
+    @test H_2 ./ H_ref ≈ ones(length(freq))
+    # ... but ours (H1) should be the better one
+    delta_1 = H_1 ./ H_ref .- 1
+    delta_2 = H_2 ./ H_ref .- 1
+    @test norm(delta_1) < norm(delta_2)
```

Comment:
I'm not sure comparing to the coefficients obtained with a 10(?)-year old Matlab version makes much sense. Isn't it sufficient to verify that ours are good?
```suggestion
    H_1 = freqresp(f, freq)
    @test H_1 ./ H_ref ≈ ones(length(freq))
```